specs/slider/slider.md

```diff
@@ -0,0 +1,189 @@
+
+- *A developer building an app with the component and interacting through HTML/CSS/JavaScript.*
+A developer can inject labels on the slider as children, if they use <slider-label> elements those will be styled a specific way internally and the dev can override any part of that style they want. Alternatively the dev can put whatever they want in children and style as they wish.
+
+- *A designer customizing the component.*
+A designer can override any internal styling applied to the slotted labels, the background track, the slider track and the thumb(s) if appropriate.
+  
+### Features
+- Implement your own lower and upper thumb UI, default provided
+- Implement your own background and slider track UI, default provided
+- Handle the `slider`'s `onValueChange` callback to be notified when the value has changed.
+- Customize the tick marks along the track
+- Include slider in form based operations
+
+### Prior Art/Examples
+- [FAST-DNA (React)](https://explore.fast.design/components/slider)
+- [Material UI](https://material-ui.com/components/slider/)
+
+---
+
+### API
+
+*Component Name*:
+- `fast-slider`
+
+*Props/Attrs*
+- `min` - minimum allowed value for the slider
+- `max` - maximum allowed value for the slider
+- `step` - limits the values of the `slider` to increments of the `step` value added to the minimum value of the 
+`slider`'s total range.  The default value is 1. The minimum and maximum values of a `slider`'s range are always valid results regardless of the `step` prop. The `step` prop is used as the value for incrementing the thumb by pressing the arrow keys.
+
+ - `value` - Allows authors to specify the initial selected range of the `slider`.  It defaults to a (step constrained) value at the midpoint on the `slider`'s total range. 
+
+*Events*
+- `change` - raise the change event for external parties to be informed of the value change.
+
+*Methods*
+- `onValueChanged` - handling the internal framework call when the value attribute is changed.
+
+*Slots*
+- `background-track` - background area encompassing the track and labels
+- `slider-track` - the horizontal or vertical track along which the thumb slides
+- `thumb` - the control the user drags along the track to change the value of the slider
+
+### Anatomy and Appearance
+**Structure:**
+
+```
+  <div
+    role="slider"
+  >
+    <input type="range" 
+      id="${x => x.id}"
+      min="${x => x.min}"
+      max="${x => x.max}"
+      step="${x => x.step}"
+      value="${x => x.value}"
+    >
+    <slot name="label">
+      <div>${x => x.label}</div>
+    </slot>
+    <slot name="background-track">
+      <div></div>
+    </slot>
+    <slot name="slider-track">
+      <div></div>
+    </slot>
+    <slot name="thumb">
+      <div></div>
+    </slot>
+    <slot></slot>
+  </div>
+```
+
+## Implementation
+
+```

```

Comment:
I think you could put "html" after the backticks to get nicer syntax highlighting.